### PR TITLE
Custom activities

### PR DESCRIPTION
**Pull request by deathtenk**
This Adds the ability to add custom activities. Either by setting the player prefs like so:

```csharp
PlayerPrefs.SetString("LRSActivityId", "http://video.games/button-clicker/level/1");
PlayerPrefs.SetString("LRSActivityDefinition", "Level 1 of button-clicker");
```

or by overloading the arguments to the publisher methods like the following:
```csharp
publisher.SendStartedStatement("http://video.games/clicker/level/1", "Level 1 of clicking game");
```

### CODE REVIEW

**Review by cliffcaseyyet — COMMENTED** [review]
This looks good, but can you add one thing I noticed? Can you put in the readme what part of an xAPI statement the individual fields map to? It was hard to parse from an integration perspective because it's (understandably) obscured.

**Comment by deathtenk** [comment]
@cliffcaseyyet addressed via https://github.com/yetanalytics/unity-xapi-publisher/pull/6/commits/b3e4f69e0d3e7a0ee9d54da18c3dccd35557ee3d and https://github.com/yetanalytics/unity-xapi-publisher/pull/6/commits/986891c09649a07c19cbc4e94d2b0f2c332127c7

**Review by cliffcaseyyet — APPROVED** [review]
